corehq/apps/app_manager/static/app_manager/js/releases/releases.js

```diff
@@ -496,6 +506,10 @@ hqDefine('app_manager/js/releases/releases', function () {
             });
         };
 
+        self.toggleReleasesControl = function (enableRelease) {
```

Comment:
Nitpick: is this needed? Is the intention that other code not access observables directly?

Reply:
Yes, that's the intention.